traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices_required_capabilities.go

```diff
@@ -281,6 +286,25 @@ func (rc *RequiredCapability) Create() (error, error, int) {
 	return nil, nil, http.StatusOK
 }
 
+func (rc *RequiredCapability) checkServerCap() (error, error, int) {
+	tx := rc.APIInfo().Tx
+
+	// Get server capability name
+	name := ""
+	if err := tx.QueryRow(`
```

Comment:
Adding this query broke the flow expected by the sqlmock statements in the unit tests

Reply:
To be clear, you should change the tests, not your code.